.github/workflows/test.yml

```diff
@@ -48,4 +48,19 @@ jobs:
 
       - name: Build
         run: cd Examples/topics && swift build
+  
+  cache-example:
+    runs-on: ubuntu-latest
+    container: swift:5.8
+    steps:
+    #   - uses: swift-actions/setup-swift@65540b95f51493d65f5e59e97dcef9629ddf11bf
+    #     with:
+    #         swift-version: ${{ matrix.swift }}
+      - name: Setup repo
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.github-token }}
+
+      - name: Build
+        run: cd Examples/cache && swift build
```

Comment:
You can just append the `Build` portion of this job to the preceding `topics-example` job so it can reuse the same setup. You'll probably want to rename the job `example` and the build steps `Build topics example` and `Build cache example`.